packages/web/src/app/web-player/WebPlayer.jsx

```diff
@@ -749,15 +749,16 @@ class WebPlayer extends Component {
                 />
                 <Route exact path={HISTORY_PAGE} component={HistoryPage} />
                 {isDevOrStaging && (
-                  <>
-                    <Route exact path={DEV_TOOLS_PAGE} component={DevTools} />
-                    <Route
-                      exact
-                      path={SOLANA_TOOLS_PAGE}
-                      component={SolanaToolsPage}
-                    />
-                  </>
+                  <Route exact path={DEV_TOOLS_PAGE} component={DevTools} />
+                )}
+                {isDevOrStaging && (
```

Comment:
wait why do both of these appear twice now

Reply:
oh wait, thought i deleted the ones below, guess i didn't commit, one sec

Reply:
also style guide we prefer `?` ternaries to `&&`